File: annotation/src/main/java/ru/bachev/annotation/ApplicationApp.java
package ru.bachev.annotation;

import org.springframework.context.support.ClassPathXmlApplicationContext;

public class ApplicationApp {
    public static void main(String[] args) {
        ClassPathXmlApplicationContext context = new ClassPathXmlApplicationContext("applicationContext.xml");
        Car car = context.getBean("audi", Car.class);
        System.out.println(car.getSpeed());
        System.out.println(car.getEngine());
        context.close();
    }
}

File: annotation/src/main/kotlin/ru/bachev/annotation/ApplicationApp.kt
package ru.bachev.annotation

import org.springframework.context.support.ClassPathXmlApplicationContext

fun main() {
    val context = ClassPathXmlApplicationContext("applicationContext.xml")
    val car = context.getBean("audi", Car::class.java)
    println(car.speed)
    println(car.engine)
    context.close()
}